Misc/NEWS.d/next/Core and Builtins/2019-09-09-15-17-58.bpo-38073.ZoKYOU.rst

```diff
@@ -0,0 +1 @@
+`:mod:`pwd` is :pep:`384` compatible
```

Comment:
sphinx doesn't like the reference here, please change the text to ``Make pwd extension module PEP-384 compatible``